pephub/routers/api/v1/schemas.py

```diff
@@ -168,26 +170,45 @@ async def create_schema_for_namespace_by_file(
         )
 
 
-@schemas.get("/{namespace}/{schema}", response_model=SchemaGetResponse)
+@schemas.get("/{namespace}/{schema}", response_model=Union[SchemaGetResponse, dict])
```

Comment:
A suggestion would be to take whats below... `/namespace/schema/file` and make that a query param...

I like the idea of keeping it in one since it seems to be an argument `?format=file`. But I will leave it to y'all to make that decision

Reply:
I wrote this way based on project endpoint (where we have similar situation), But we can change it if needed

Reply:
Nah, I think its fine 👍🏼